### ISSUE TITLE
Notifications not working

### ISSUE THREAD

**Issue by Cliffy786** [aritchie/geofences#10]
Hi @aritchie  

First off I'd like to say thank you for the plugin. I've been playing around with it and managed to get the geofences set up, basically everything is working fine except for the notifications, Here's some of the code I've been playing around with:

 ```
geofences.StartMonitoring(new GeofenceRegion("48 Silver Ave", new Plugin.Geofencing.Position(-29.8344948, 31.0217101), Distance.FromMeters(300)));
                //geofences.StartMonitoring(new GeofenceRegion("Skyy Cafe", new Plugin.Geofencing.Position(-29.834615, 31.022979), radius));
                geofences.StartMonitoring(new GeofenceRegion("22 Clancy Ave", new Plugin.Geofencing.Position(-29.818346, 31.006844), radius));
                Debug.WriteLine($"Regions currently monitered: {geofences.MonitoredRegions.Count}");
                int num = geofences.MonitoredRegions.Count;
                var region = geofences.MonitoredRegions.First();
                var result = await geofences.RequestState(new GeofenceRegion("48 Silver Ave", new Plugin.Geofencing.Position(-29.8344948, 31.0217101), Distance.FromMeters(400)), null).ConfigureAwait(false);
                var result2 = await geofences.RequestState(new GeofenceRegion("22 Clancy Ave Bru", new Plugin.Geofencing.Position(-29.818346, 31.006844), Distance.FromMeters(80)), null).ConfigureAwait(false);

                await CrossNotifications.Current.Send(new Notification
                {
                    Title = "Loyalty Worx",
                    Message = "Hey! watkine"
                });
```
Now the thing is, I was wondering why there were no notifications when I entered geofences so I added the five lines of code just to fire a sample notification instead of just having it in the App.cs, upon doing so there was an exception thrown and the message was "System.NotSupportedException: Cannot update DbSettings: it has no PK".

**Comment by Cliffy786** [comment]
Good day @aritchie Sorry about that my bad I was wrong in posting the issue in the wrong plugin repository, I only later on realized that I should have posted the issue in the other repository, so stupid of me. I'm sorry for not posting all the details requested such as the OS Version, stack trace and the plugin version very incompetent on my behalf. In the future I will make sure to pay attention to the finer details.

-That being said, The device I'm debugging on is an android kit kat device. 
-The plugin version was v4.0.0 initially when the exception occured, but then I upgraded to 4.0.2 but the 
 same exception was thrown
- Here's the stack trace, there's not much though(I copied the whole stack trace)

`at SQLite.SQLiteConnection.Update (System.Object obj, System.Type objType) [0x00039] in /Users/vagrant/git/src/SQLite.cs:1474 \n  at SQLite.SQLiteConnection.Update (System.Object obj) [0x00005] in /Users/vagrant/git/src/SQLite.cs:1445 \n  at Plugin.Notifications.NotificationsImpl.Send (Plugin.Notifications.Notification notification) [0x00034] in C:\\dev\\acr\\notifications\\Plugin.Notifications.Android\\NotificationsImpl.cs:46 \n  at LoyaltyWorx.Page1+<LoadData>d__4.MoveNext () [0x0049c] in C:\\dev\\Xamarin Projects\\LoyaltyWorx\\LoyaltyWorx\\LoyaltyWorx\\Page1.xaml.cs:59
`
